src/app/@ansyn/map-facade/components/imagery-container/imagery-container.component.ts

```diff
@@ -1,10 +1,12 @@
-import { Component, Input } from '@angular/core';
+import { Component, Inject, Input } from '@angular/core';
 import { Store } from '@ngrx/store';
 import { ActiveImageryMouseEnter, ActiveImageryMouseLeave, SynchronizeMapsAction } from '../../actions/map.actions';
 import { CaseMapState } from '@ansyn/core/models/case.model';
 import { Overlay } from '@ansyn/core/models/overlay.model';
 import { IMapState, mapStateSelector } from '@ansyn/map-facade/reducers/map.reducer';
-import { Observable } from 'rxjs';
+import { Observable } from 'rxjs/Observable';
```

Comment:
revert to "import { Observable } from 'rxjs';"

Reply:
done